docker/pytest/requirements.in

```diff
@@ -1,3 +1,7 @@
+# IMPORTANT:
+# In order for requirements.txt to be correct you must follow the
+# playbook entry: go/cobalt/team/...
```

Comment:
Could we publish the playbook perhaps, in a `.md` doc somewhere in the repo ?

Reply:
I boiled down the instructions I'm adding in cl/648777889 to 4 lines and put them in each file instead.